buildpack/nginx.py

```diff
@@ -19,6 +21,35 @@
 }
 
 
+# Fix for Chrome SameSite enforcement (from Chrome 80 onwards)
+# Runtime will set this cookie in runtime versions >= SAMESITE_COOKIE_WORKAROUND_LESS_MX_VERSION
+SAMESITE_COOKIE_WORKAROUND_ENV_KEY = "SAMESITE_COOKIE_WORKAROUND"
```

Comment:
Should we call it "workaround"? Since this is a customer-facing environment option, it may not have the right ring to it. Perhaps "SAMESITE_COOKIE_PRE_MX812".

Reply:
I agree, will change this